src/dbus.c

```diff
@@ -487,6 +511,10 @@ method_config_apply(sd_bus_message *m, void *userdata, sd_bus_error *ret_error)
     d->config_dirty = g_strdup(d->config_id);
 
     if (d->try_pid < 0) {
+        r = _backup_global_state(ret_error);
+        if (r < 0)
+            return r; // LCOV_EXCL_LINE
```

Comment:
nit: it would have been nice to switch the last 2 commits

I squinted really hard at the code in the previous commit trying to understand where those backup files came from ;-)

Reply:
Yeah.. It was kind of an afterthough, after realizing that the `io.netplan.Netplan.Config.Try()` API does not work as expected. Sorry for that. :)